src/kubearchive/conditional-checks.ts

```diff
@@ -0,0 +1,17 @@
+import { createConditionsHook } from '~/feature-flags/hooks';
+import { ensureConditionIsOn } from '~/feature-flags/utils';
+import { commonFetch } from '~/k8s';
+import { KUBEARCHIVE_PATH_PREFIX } from './const';
+
+export const checkIfKubeArchiveIsEnabled = async () => {
+  try {
+    await commonFetch('/livez', { pathPrefix: KUBEARCHIVE_PATH_PREFIX });
+    return true;
```

Comment:
Need to check if we throw error from commonFetch or should I return `response.ok`